test/Nancy.Tests.MSBuild/Nancy.Tests.csproj

```diff
@@ -276,15 +276,15 @@
     <Compile Include="..\Nancy.Tests\Unit\Helpers\HttpUtilityFixture.cs">
       <Link>Unit\Helpers\HttpUtilityFixture.cs</Link>
     </Compile>
-    <Compile Include="..\Nancy.Tests\Unit\HttpLinkBuilderFixture.cs">
-      <Link>Unit\HttpLinkBuilderFixture.cs</Link>
-    </Compile>
-    <Compile Include="..\Nancy.Tests\Unit\HttpLinkFixture.cs">
-      <Link>Unit\HttpLinkFixture.cs</Link>
-    </Compile>
-    <Compile Include="..\Nancy.Tests\Unit\HttpLinkRelationFixture.cs">
-      <Link>Unit\HttpLinkRelationFixture.cs</Link>
-    </Compile>
+    <Compile Include="..\Nancy.Tests\Unit\HttpLinkBuilderFixture.cs">
+      <Link>Unit\HttpLinkBuilderFixture.cs</Link>
+    </Compile>
+    <Compile Include="..\Nancy.Tests\Unit\HttpLinkFixture.cs">
+      <Link>Unit\HttpLinkFixture.cs</Link>
+    </Compile>
+    <Compile Include="..\Nancy.Tests\Unit\HttpLinkRelationFixture.cs">
+      <Link>Unit\HttpLinkRelationFixture.cs</Link>
+    </Compile>

```

Comment:
This was due to inconsistent line breaks. These lines had LF and rest of the file has CRLF.

I can remove this change from the commit if you prefer to keep it pure.